### PR TITLE
Fix dates in warn03

### PR DESCRIPTION
**Pull request by zimeon**
To be valid the version dates must not be empty

### CODE REVIEW

File: 1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "bb123cd4567",
-  "head": "v0002",
```
Comment (julianmorley) [inline 1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json]: v0002 is correct; this is the inventory as it was when v0002 was the most recent version.

File: 1.0/warn-objects/warn03_zero_padded_versions/v0003/inventory.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "id": "bb123cd4567",
-  "head": "v0003",
+  "head": "v0004",
```
Comment (julianmorley) [inline 1.0/warn-objects/warn03_zero_padded_versions/v0003/inventory.json]: Same as above; this is the inventory that was current when v0003 was most recent version.

File: 1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json (hunk 1 above)
Comment (zimeon) [reply 1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json]: Yup, my bad. I create old version inventories by editing the last one down and forget to change the `head`.... will fix

File: 1.0/warn-objects/warn03_zero_padded_versions/v0003/inventory.json (hunk 2 above)
Comment (zimeon) [reply 1.0/warn-objects/warn03_zero_padded_versions/v0003/inventory.json]: Addressed by 1d14868

File: 1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json (hunk 1 above)
Comment (zimeon) [reply 1.0/warn-objects/warn03_zero_padded_versions/v0002/inventory.json]: Addressed by 1d14868

**Comment by zimeon** [comment]
Per spec `created` must be a valid datetime (`message` can be empty)

**Pull request merged**